spec/components/schemas/reports/message/entry/base.ts

```diff
@@ -0,0 +1,43 @@
+// tslint:disable:max-line-length
+import { SchemaObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../../utils/ref';
+
+const base: SchemaObject = {
+  title: 'Entry object for Message Report',
+  description: 'This is a entry object model for Message Report.',
+  type: 'object',
+  properties: {
+    channel: {
+      title: 'Channel',
+      type: 'string',
+      readOnly: true,
+    },
+    type: {
+      title: 'Type of message',
+      description: 'Type of message, that may be either "message" or "notification"',
```

Comment:
Também não lembro como está nos outros modelos, mas faltaram artigos, deveria ser "The type of the message".

"that" não tá errado, mas acho que "which" fica melhor.

Ponto final também não faria mal.

Reply:
Ah, e notei também que message tá no singular, mas como as entradas são agrupamentos de mensagens, deveria estar no plural.